src/GoogleAdWordsEventForwarder.js

```diff
@@ -393,6 +553,22 @@
                     return 'Can\'t initialize forwarder: ' + name + ', conversionId is not defined';
                 }
 
+                if (window.gtag && forwarderSettings.enableGtag == 'True') {
```

Comment:
Shouldn't matter in this case, but
```suggestion
                if (window.gtag && forwarderSettings.enableGtag === 'True') {
```